DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/Model/DicomAnonymizationErrorCode.cs

```diff
@@ -0,0 +1,18 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core.Model
+{
+    public enum DicomAnonymizationErrorCode
+    {
+        MissingConfigurationFields,
+        InvalidConfigurationValues,
+        UnsupportedAnonymizationRule,
+        MissingRuleSettings,
```

Comment:
Are we using this variable anywhere?

Reply:
Thanks, fixed it. Add this error code in one of the exception